tests/conflicts.rs

```diff
@@ -5,14 +5,14 @@ use clap::{App, Arg, ArgGroup, ErrorKind};
 static CONFLICT_ERR: &str = "error: The argument '-F' cannot be used with '--flag'
 
 USAGE:
-    clap-test <positional> <positional2> --flag --long-option-2 <option2>
+    clap-test <positional> --long-option-2 <option2> -F
 
 For more information try --help";
 
-static CONFLICT_ERR_REV: &str = "error: The argument '--flag' cannot be used with '-F'
+static CONFLICT_ERR_REV: &str = "error: The argument '-F' cannot be used with '--flag'
 
 USAGE:
-    clap-test <positional> <positional2> -F --long-option-2 <option2>
```

Comment:
@CreepySkeleton Do we want all the positionals?

Reply:
Yeah, we do

Reply:
They are supposed to be there, yes. I don't realize why the code isn't working; I imagine it should do well. Will dive in it  in several hours.